style.css

```diff
@@ -150,112 +150,108 @@
   text-decoration: none;
   padding-inline-start: 45px;
 }
-
-.header-left__nav li {
+.nav__item {
   text-transform: uppercase;
   list-style-type: none;          
   margin-right: 35px;
   margin-bottom: 15px;
 }
 
-.header a {
+.nav__link
+,
+.sign-in__link
+,
+.cart__link {
   color: black;
-  text-decoration: none;          
-  font-family: montserratbold;
-  font-size: 12px;
-  line-height: 11px;
-}
-
-.header-right__search, .header-right__search_sign-in, .header-right__search_cart {
+  text-decoration: none;      
+  font: bolder 12px/11px Gotham Pro;
+}
+.search-form
+, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5483930)